source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/ACRCustomSubmitTargetBuilder.h

```diff
@@ -5,13 +5,9 @@
 //  Copyright © 2020 Microsoft. All rights reserved.
 //
 
-#import <Foundation/Foundation.h>
 #import <AdaptiveCards/AdaptiveCards.h>
+#import <Foundation/Foundation.h>
```

Comment:
maybe @jwoo-msft knows more, but in UWP the order of `#include` (not `#import`) is important and shouldn't be sorted (order matters).

I know that the current `.clang-format` disables `#include` sorting, but does order matter in `#import` directives in ObjC?

Reply:
no, I never encountered issues after changing the order.